closed/autoconf/custom-hook.m4

```diff
@@ -75,6 +75,11 @@ AC_DEFUN([OPENJ9_CONFIGURE_CMAKE],
   # at this point with_cmake should either be no, or the name of the cmake command
   if test "x$with_cmake" = xno ; then
     OPENJ9_ENABLE_CMAKE=false
+
+    # Currently, mixedrefs mode is only available with CMake enabled
+    if test "x$OPENJ9_ENABLE_MIXED_REFERENCES" = xtrue ; then
+      AC_MSG_ERROR([--with-mixedrefs cannot be used if CMake is disabled])
```

Comment:
Can you make this message instructive, perhaps, '--with-mixedrefs requires --with-cmake`?